packages/slate-react/src/hooks/use-editor.tsx

```diff
@@ -1,16 +1,12 @@
-import { createContext, useContext } from 'react'
+import { useContext } from 'react'
 
-import { ReactEditor } from '../plugin/react-editor'
+import { EditorContext } from './use-slate-static'
 
-/**
- * A React context for sharing the editor object.
- * @deprecated Use useSlateStatic instead.
- */
-
-export const EditorContext = createContext<ReactEditor | null>(null)
+export { EditorContext }
```

Comment:
Why exporting `EditorContext` again?  We're importing it already from `'./use-slate-static`.  Otherwise, looks ok.

Reply:
make sense, no need export again. will fix in 1 minute.

Reply:
Looks good!